src/libs/lib_api.h

```diff
@@ -86,6 +86,7 @@ OPTIONAL(int, set_params, struct dt_lib_module_t *self, const void *params, int
 OPTIONAL(void, init_presets, struct dt_lib_module_t *self);
 OPTIONAL(void, manage_presets, struct dt_lib_module_t *self);
 OPTIONAL(void, set_preferences, void *menu, struct dt_lib_module_t *self);
+OPTIONAL(int, preset_autoapply, struct dt_lib_module_t *self);
```

Comment:
seems like this should return a gboolean as it seems to return a status whether it support auto-apply a preset, right? So better return TRUE or FALSE.

Reply:
I'm not sure about changing the definition here : none of the other modules functions return a gboolean...
But I agree that when defined, the function themselves can return TRUE or FALSE which will be interpreted as int at the end.

Reply:
That's correct, but we have started since some time to change routines to use gboolean when possible. At some point all of them should be converted.